trajectory_publisher/launch/trajectory_setpointraw.launch

```diff
@@ -29,26 +9,4 @@
         <param name="Kpos_z" value="30.0" />
         <param name="reference_type" value="16" />
   </node>
-
-  <include file="$(find mavros)/launch/node.launch">
-      <arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
-      <arg name="config_yaml" value="$(find mavros)/launch/px4_config.yaml" />
-
-      <arg name="fcu_url" value="$(arg fcu_url)" />
-      <arg name="gcs_url" value="$(arg gcs_url)" />
-      <arg name="tgt_system" value="$(arg tgt_system)" />
-      <arg name="tgt_component" value="$(arg tgt_component)" />
-      <arg name="log_output" value="$(arg log_output)" />
-      <arg name="fcu_protocol" value="$(arg fcu_protocol)" />
-      <arg name="respawn_mavros" default="$(arg respawn_mavros)" />
-  </include>
-
-  <include file="$(find px4)/launch/posix_sitl.launch">
-      <arg name="vehicle" value="$(arg mav_name)"/>
-  </include>
-  
-  <group if="$(arg visualization)">
-      <node type="rviz" name="rviz" pkg="rviz" args="-d $(find geometric_controller)/launch/config_file.rviz" />
-  </group>
-
```

Comment:
If this was deleted to run in the real system, could you name this file as `mav_trajectory_setpointraw.launch`  and the `trajectory_setpointraw_orig.launch` as `sitl_trajectory_setpointraw.launch` . I think this fits with the launchfile naming convenion of the `geometric_controller` better